src/protagonist/API/Features/Image/Validation/HydraImageValidator.cs

```diff
@@ -111,4 +111,16 @@ private void DeliveryChannelDependantValidation()
             .WithMessage(
                 $"ImageOptimisationPolicy '{KnownImageOptimisationPolicy.UseOriginalId}' only valid for image delivery-channel");
     }
+
+    private bool DeliveryChannelIsValidForMediaType(string channel, string mediaType)
+    {
+        try
+        {
+            return AssetDeliveryChannels.IsChannelValidForMediaType(channel, mediaType);
+        }
+        catch(ArgumentOutOfRangeException)

```

Comment:
Would it be cleaner to optionally throw an exception in `IsChannelValidForMediaType` if we know that it could be invalid in some instances?

```cs
static bool IsChannelValidForMediaType(string deliveryChannel, string mediaType);

static bool IsChannelValidForMediaType(string deliveryChannel, string mediaType, bool throwIfUnknownChannel = true);
```

Reply:
I think it would be. It saves on similar handlers needing to be written elsewhere if needed, too.